components/dictionary.txt

```diff
@@ -346,6 +346,7 @@ Satija
 Sca
 scater
 SCE
+ScPCA
```

Comment:
At some earlier point we loosely decided to not include in dictionary but keep as code format (as in `ScPCA` so spell check ignores it) - 
https://github.com/AlexsLemonade/training-modules/pull/579#discussion_r1036261511